frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketsTopPanel/ticketsTopPanel.component.tsx

```diff
@@ -94,7 +93,7 @@ export const TicketsTopPanel = ({
 					/>
 					{_.get(formState.errors, `properties.${BaseProperties.DESCRIPTION}`) && <ErrorTextGap />}
 				</DescriptionProperty>
-				{hasIssueProperties && <IssuePropertiesInputs onBlur={onPropertyBlur} readOnly={readOnly} />}
+				<IssuePropertiesInputs properties={properties} onBlur={onPropertyBlur} readOnly={readOnly} />
```

Comment:
Im not sure about the naming of `IssuePropertiesInputs` because now is not only for issue properties.
I see two options.
- rename it to another thing that is more representative (sorry I cant imagine of another name)
- refactor `Issuepropertisinputs` to reuse a `statusinput` component and keep the 'hasissueProperties' here to render that one.  and have a plain `statusPropertyBar` that would reuse `statusinput` ( `<FlexContainer> <StatusInput /> </FlexContainer>`  )  that would render if hasissueProperties is false